www.chuyu.me/es.xml

```diff
@@ -34,6 +34,6 @@
   <String Src="%ChuyuTream%" Dst="&amp;nbsp;2017&amp;nbsp;Equipo de Chuyu"/>
   <String Src="%ICP%" Dst=""/>
   <String Src="%Document%" Dst="Documentación de Dism++"/>
-  <String Src="%logs%" Dst="Cambios"/>
+  <String Src="%logs%" Dst="Registro de cambios"/>

```

Comment:
The correct thing would be to display everything without cutting on the page, enter to display all the complete words.

Reply:
Thanks, but  I prefer to be dapper, the new  translations are too long.

Reply:
Some parts are poorly translated when expressing a function on the page.

example: Line 16: "Limpiar WinSxS"  -------->it looks bad
Correct: "Limpieza de WinSxS"

otherwise if you like it well I can not do anything.

Reply:
How bad that you accept something well done.

Reply:
OK，I adjusted "Limpiar WinSxS" now.
Thanks Jhovany200